sihl-queue/src/admin_ui.ml

```diff
@@ -162,46 +185,191 @@ let table scope csrf (jobs : Sihl.Contract.Queue.instance list) =
+}
+
+.sihl-admin-ui-queue-table-header {
+    background-color: #DDDDDD;
+}
+
+.sihl-admin-ui-queue-table-body-row-status-failed {
+    background-color: #FFCCCC;
+}
+
+.sihl-admin-ui-queue-table-body-row-status-succeeded {
+    background-color: #CCFFCC;
+}
+
+.sihl-admin-ui-queue-table-button {
+    border: 1px solid black;
+}
+.sihl-admin-ui-queue-table-button:hover{
+    color: #FFFFFF;
+    background-color: #777777;
+}
+|}]
+;;
+
+let dark =
+  [%html
+    {|
+body {
+    background-color: #282828;
+}
+
+.sihl-admin-ui-queue-back {
+    color: white;
+}
+
+.sihl-admin-ui-table-header-cell, .sihl-admin-ui-table-body-cell {
+    border: 1px solid #282828;
+    color: #D8D8D8;
+}
+
+.sihl-admin-ui-queue-table-header {
+    background-color: #404040;
+}
+
+.sihl-admin-ui-queue-table-body-row-status-failed {
+    background-color: #5d3030;
+}
+
+.sihl-admin-ui-queue-table-body-row-status-succeeded {
+    background-color: #305430;
+}
+
+.sihl-admin-ui-queue-table-button {
+    border: 1px solid #282828;
+    background-color: #282828;
+    color: white;
+}
+.sihl-admin-ui-queue-table-button:hover{
+    color: black;
+    background-color: #EEEEEE;
+}
+|}]
+;;
+
+let page ?back ?theme body =
   let body =
     match back with
     | Some back ->
-      let back_button = [%html {|<a href="|} back {|">← Go back</a>|}] in
+      let back_button =
+        [%html
+          {|<a href="|}
+            back
+            {|" class="sihl-admin-ui-back sihl-admin-ui-queue-back">← Go back</a>|}]
+      in
       List.cons back_button body
     | None -> body
   in
+  let body =
+    match theme with
+    | Some `Light ->
+      let theme = [%html {|<style>|} [ base; light ] {|</style>|}] in
+      List.concat [ body; [ theme ] ]
+    | Some `Dark ->
+      let theme = [%html {|<style>|} [ base; dark ] {|</style>|}] in
+      List.concat [ body; [ theme ] ]
+    | Some (`Custom _) -> body
+    | None ->
+      let theme = [%html {|<style>|} [ base; light ] {|</style>|}] in
+      List.concat [ body; [ theme ] ]
+  in
   let body =
     match Sihl.Configuration.read_string "HTMX_SCRIPT_URL" with
     | Some htmx ->
       let htmx_script = [%html {|<script src="|} htmx {|"></script>|}] in
       List.concat [ body; [ htmx_script ] ]
     | None -> body
   in
-  [%html
-    {|
-<!doctype html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8"/>
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link href="/assets/styles.css" rel="stylesheet">
-    <title>Hello world!</title>
-  </head>
-    <body>|}
-      body
+  match theme with
+  | Some (`Custom url) ->
+    [%html
       {|
-     </body>
-</html>
-|}]
+       <!doctype html>
+       <html lang="en">
+         <head>
+           <meta charset="UTF-8"/>
+           <meta name="viewport" content="width=device-width, initial-scale=1">
+           <link href="|}
+        url
+        {|" rel="stylesheet">
+           <link href="/assets/styles.css" rel="stylesheet">
```

Comment:
I think this can be removed since a custom style link can be provided now.

Reply:
Done

Reply:
Thanks!